target/linux/ramips/image/mt76x8.mk

```diff
@@ -240,6 +240,15 @@ define Device/jotale_js76x8-32m
 endef
 TARGET_DEVICES += jotale_js76x8-32m
 
+define Device/kroks_kndrt31r19
+  IMAGE_SIZE := 14m
+  DEVICE_VENDOR := Kroks
+  DEVICE_MODEL := Rt-Pot mXw DS RSIM
+  $(Device/Kroks-mpci)
+  SUPPORTED_DEVICES += kndrt31r19
```

Comment:
What's this line required for?

Reply:
I'm not sure but it might be needed to be able to flash from stock firmware.
I also saw this on other ports to master.

Reply:
Yes, but in the other ports it's for sysupgrade from older versions of OpenWrt.

Is the OEM firmware OpenWrt?

Reply:
Hmm, good question.
Judging from https://github.com/kroks-free/openwrt, it mostly is. They have some proprietary modules on top, but overall it looks and feels as OpenWrt clone.
I'll try to upgrade with sysupgrade and we'll see.

Reply:
Sysupgrade went well via vendor's WEB UI, so this line is probably needed so the image could be accepted.